JS/main.js

```diff
@@ -3,15 +3,22 @@
 const toDoInput = document.querySelector('.todo-input');
 const toDoBtn = document.querySelector('.todo-btn');
 const toDoList = document.querySelector('.todo-list');
+const standardTheme = document.querySelector('.standard-theme');
+const lightTheme = document.querySelector('.light-theme');
+const darkerTheme = document.querySelector('.darker-theme');
 
 
 // Event Listeners
 
 toDoBtn.addEventListener('click', addToDo);
 toDoList.addEventListener('click', deletecheck);
 document.addEventListener("DOMContentLoaded", getTodos);
+standardTheme.addEventListener('click', () => changeTheme('standard'));
+lightTheme.addEventListener('click', () => changeTheme('light'));
+darkerTheme.addEventListener('click', () => changeTheme('darker'));
 
-
+// Global variable to record theme status, initialized at default value:
+let chosenTheme = 'standard';
```

Comment:
Is there a way, to retain the theme, once reloaded? Let me know if it is possible, just curious.

Reply:
Yes, I added this feature using the localStorage property as you did for the todo items.

Reply:
Looks good. 👍🏻